test/integration/client-side-encryption/client_side_encryption.prose.test.js

```diff
@@ -1426,6 +1434,291 @@ describe('Client Side Encryption Prose Tests', metadata, function () {
     });
   });
 
+  context('12. Explicit Encryption', eeMetadata, function () {
+    const data = path.join(__dirname, '..', '..', 'spec', 'client-side-encryption', 'etc', 'data');
+    let encryptedFields;
+    let key1Document;
+    let key1Id;
+    let setupClient;
+    let keyVaultClient;
+    let clientEncryption;
+    let encryptedClient;
+
+    beforeEach(async function () {
+      const mongodbClientEncryption = this.configuration.mongodbClientEncryption;
+      // Load the file encryptedFields.json as encryptedFields.
+      encryptedFields = EJSON.parse(
```

Comment:
Data for the `encryptedFields` option and key document are now provided in 2 new files in the etc/ directory in the csfle spec directory.